internal/controller/utils.go

```diff
@@ -66,6 +60,21 @@ func MatchAnySecretOfTypeFilter(secretType corev1.SecretType, parentFunc control
 	return SimpleFilter(isSecretOfType, parentFunc)
 }
 
+func MatchAnySecretOfTypesFilter(secretTypes []corev1.SecretType, parentFunc controllerlib.ParentFunc, namespaces ...string) controllerlib.Filter {
```

Comment:
Can the implementation of `MatchAnySecretOfTypeFilter` above be changed to simply be one line which calls this new `MatchAnySecretOfTypesFilter`?

Reply:
will be done in next push